PIL/Image.py

```diff
@@ -599,7 +635,7 @@ def _dump(self, file=None, format=None, **options):
             self.save(file, format, **options)
         return file
 
-    def __eq__(self, other):
+    def __eq__(self, other):  # type: ignore # type: (Image) -> bool
         return (self.__class__.__name__ == other.__class__.__name__ and
```

Comment:
I think the problem I read about was the return type (for some reason), rather than the parameter, but we can certainly look at this again.